o We need to include libnsl as a dependency for libpwdb if libpwdb is static.
--- Linux-PAM-0.77/configure.in	2003-06-30 23:08:37.000000000 -0400
+++ Linux-PAM-0.77/configure.in	2003-06-30 23:08:32.000000000 -0400
@@ -224,9 +224,15 @@
 AC_CHECK_LIB(nsl, yp_maplist, HAVE_LIBNSL=yes ; AC_DEFINE(HAVE_LIBNSL),
 	HAVE_LIBNSL=no)
 AC_SUBST(HAVE_LIBNSL)
+
+if test $HAVE_LIBNSL = yes ; then
+	pwdblibs="$pwdblibs -lnsl"
+fi
 AC_CHECK_LIB(pwdb, pwdb_db_name, HAVE_LIBPWDB=yes ; AC_DEFINE(HAVE_LIBPWDB),
-	HAVE_LIBPWDB=no)
+	HAVE_LIBPWDB=no,$pwdblibs)
 AC_SUBST(HAVE_LIBPWDB)
+unset pwdblibs
+
 AC_CHECK_LIB(fl, yywrap, HAVE_LIBFLEX=yes ; AC_DEFINE(HAVE_LIBFLEX),
 	HAVE_LIBFLEX=no)
 AC_SUBST(HAVE_LIBFLEX)
